src/js/WCartesianChart.js

```diff
@@ -512,8 +512,8 @@ WT_DECLARE_WT_MEMBER_BIG
         for (i = 0; i < len(sliders()); ++i) {
           var o = $('#' + sliders()[i]);
           if (o) {
-            var sobj = o.data('sobj');
-            if (sobj && sobj.xAxis === xAx) {
+            var sobj = o[0].WtSObj;
```

Comment:
Typo? This should be `wtSObj`.

In combination with the other remark, this would become:

```cpp
var sobj = o.wtSObj;
```